src/main/ipc.js

```diff
@@ -161,8 +161,13 @@ function init (cwd, state) {
     main.send('render', state)
   }
 
-  if (state.saved.credentials) {
```

Comment:
But how the ... fucking javascript

Reply:
Problem is that state.saved.credentials is sometimes `null`, sometimes `{}`. We need to make sure that it's not null, it's a object and it's not empty. Maybe this can be done easier by preventing it from getting an empty object so that we only have to check if it's not null.